merino-integration-tests/src/suggest.rs

```diff
@@ -1,6 +1,9 @@
 //! Tests Merino's ability to make basic suggestions.
-#![cfg(test)]
+// Note, setting the following line will cause a compile error. This may be due to the
+// fact that the proc_macro may not be able to locate items.
+//#![cfg(test)]
 
+#![allow(unused_imports)]
```

Comment:
What import is unused, and if so why do we need it?

Reply:
There are a number, including:
```
crate::utils::test_tools::TestReqwestClient,
crate::TestingTools
anyhow::Result
merino_settings::providers::*
request::StatusCode
std::collections::*
```

As noted in the comment above, setting `#![cfg(test)]` causes the compile to fail because `merino-integration-tests::utils::test_tools::merino_test` cannot find `crate::suggest`, presumably because `crate::suggest` is not compiled as part of the non test build. So, instead, I enabled suggest, with the assumption that the remaining, non-test function `setup_remote_settings_collection(...)` is fine being part of the release build.